docs/azure.rst

```diff
@@ -0,0 +1,74 @@
+Azure
+======
+
+************
+Blob Storage
+************
+
+========
+Overview
+========
+Azure Blob Storage is a cloud file storage system. It uses storage accounts to organize containers (similar to
+"buckets" for other storage providers) in which to store arbitrary files referred to as blobs.
+
+This connector currently only implements block blobs and not page or append blobs.
+
```

Comment:
I apologize for not catching this on the first pass. We are trying to add a new section to our documentation for each connector around how to get account information. Could you add a small blurb and / or a link on how to get credentials for Azure?

Reply:
Should be updated now! There are a lot of different ways of generating credentials so I linked out for most of the details